install.sh

```diff
@@ -90,6 +65,14 @@ echo "Created $(docker volume create --name=sentry-kafka)."
 echo "Created $(docker volume create --name=sentry-clickhouse)."
 echo "Created $(docker volume create --name=sentry-symbolicator)."
 
+# Ensure nothing is working while we install/update
+docker-compose down
```

Comment:
Do we really need "down" (removes volumes, images, etc.), and not "stop" here?

Reply:
Good question! I'd rather recreate containers but I think that automatically happens after a `docker-compose build` when needed so I think `stop` is better here. Updating.